src/cli-main.ts

```diff
@@ -701,5 +763,43 @@ async function createNewActivation(ewId: string, network: string, versionId: str
   else {
     cliUtils.logAndExit(1, `ERROR: Activation record was not able to be created for EdgeWorker Id ${ewId}, version: ${versionId} on network: ${network}!`);
   }
+}
+
+async function createAuthToken(secretKey: string, path: string, expiry: number, isACL: boolean) {
+
+  // Time calculations
+  const startTime = Math.floor(Date.now() / 1000);
+  const endTime = startTime + (expiry * 60);
+
+  var acl=path;
+  var field_delimiter = "~";
+  var new_token = "";
+  var hmac_token = "";
 
+  new_token += `st=${startTime}`;
+  new_token += field_delimiter;
+  new_token += `exp=${endTime}`;
+
+  if(isACL) {
+    new_token += field_delimiter;
+    new_token += `acl=${acl}`;
+    hmac_token = new_token;
+  } else {
+    hmac_token = new_token + field_delimiter + "url=" + escape(path);
+  }
+
+  //console.log("Token: " + new_token);
+  //console.log("HMAC signing string: " + hmac_token);
+
+  const hexedSecretKey = CryptoJS.enc.Hex.parse(secretKey);
+  const hash = CryptoJS.HmacSHA256(hmac_token, hexedSecretKey);
+  const hashStr = CryptoJS.enc.Hex.stringify(hash);
+  var auth_token = new_token + field_delimiter + `hmac=${hashStr}`;
+
+  console.log("\nAdd the following request header to your requests to get additional trace information.\n");
```

Comment:
Please use the appropriate log and exit function here so automation can still work here.  Console.log should be removed.

Reply:
Changing this to use `cli.logWithBorder() `and `cli.logWithExit()`